src/ragger/conftest/base_conftest.py

```diff
@@ -170,6 +171,17 @@ def create_backend(root_pytest_dir: Path, backend_name: str, firmware: Firmware,
 def backend(root_pytest_dir, backend_name, firmware, display, log_apdu_file, cli_user_seed):
     with create_backend(root_pytest_dir, backend_name, firmware, display, log_apdu_file,
                         cli_user_seed) as b:
+        if backend_name.lower() == "ledgerwallet" and conf.OPTIONAL.APP_NAME:
```

Comment:
I'm starting to wonder why we keep `ledgercomm` as we never use it and don't want to maintain both... I guess we'll let ledgercomm live until we pull the plug one day

Reply:
Yep agreed, but dropping it is not for this PR :)